mne_bids/mne_bids.py

```diff
@@ -57,18 +59,25 @@ def _channels_tsv(raw, fname, verbose):
 
     """
     map_chs = defaultdict(lambda: 'OTHER')
-    map_chs.update(grad='MEGGRAD', mag='MEGMAG', stim='TRIG', eeg='EEG',
-                   ecog='ECOG', seeg='SEEG', eog='EOG', ecg='ECG', misc='MISC',
-                   resp='RESPONSE', ref_meg='REFMEG')
+    map_chs.update(meggradaxial='MEGGRADAXIAL',
+                   megrefgradaxial='MEGREFGRADAXIAL',
+                   meggradplanar='MEGGRADPLANAR',
+                   megmag='MEGMAG', megrefmag='MEGREFMAG',
+                   eeg='EEG', misc='MISC', stim='TRIG',
+                   ecog='ECOG', seeg='SEEG', eog='EOG', ecg='ECG')
     map_desc = defaultdict(lambda: 'Other type of channel')
```

Comment:
`map_desc = defaultdict(lambda: 'Other type of channel') `

I think this should be changed to `'OTHER'` instead of `Other type of channel`